config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,27 @@
 
+def build_endpoint_url ep, config
+  # There is currently no tlsPort or anything similar in clowder-common-ruby
+  return URI::HTTPS.build(host: ep&.hostname, port: ep&.tlsPort).to_s if config.tls
+  return URI::HTTP.build(host: ep&.hostname, port: ep&.port).to_s
```

Comment:
I don't think rubocop is working on this file :disappointed: can you please run it manually and at least fix the issues here? It will probably blow up on the rest of the code, but we can fix it in a followup, just please don't introduce more issues.

Reply:
Yeah, I didn't notice rubocop wasn't checking this file 🙁 I fixed the issues for my changes though!